plugin/storage/integration/integration.go

```diff
@@ -422,4 +477,10 @@ func (s *StorageIntegration) IntegrationTestAll(t *testing.T) {
 	t.Run("GetLargeSpans", s.testGetLargeSpan)
 	t.Run("FindTraces", s.testFindTraces)
 	t.Run("GetDependencies", s.testGetDependencies)
+	// Remove the below line check once we have sampling store implemented for all the backend
+	if os.Getenv("STORAGE") != "cassendra" || os.Getenv("STORAGE") != "in-memory" {
```

Comment:
this is not how we exclude tests. When a backend does not support sampling, it cannot populate `s.SamplingStore`, so the test can simply check for this being nil and call Skip with appropriate message.